catalog/README.md

```diff
@@ -0,0 +1,19 @@
+# Catalog
+
+This package holds i18n for DBaaS Controller.
+
+[gotext](https://pkg.go.dev/golang.org/x/text@v0.3.3/cmd/gotext?tab=doc)
+
+To translate, we need to follow the next steps.
+
+- Run `make gen` to collect all messages from source code; it also merges them with existing translation in `messages.gotext.json` and writes them into `out.gotext.json`.
+- As a result, we will receive all new (not translated messages) and all old (translated messages from `messages.gotext.json`) in `out.gotext.json`.
+- In the next step, we need to copy `out.gotext.json` into messages.gotext.json and fill in all required translation values.
```

Comment:
Why we need to copy it manually? Can we automate this?
Do we also need to keep both files?

Reply:
Good point.
I don't have a "right" answer for this question, I did not find any best practice for this tool as it is relatively new, and our case is particular - translation from "English" to English.
When I experimented with the translation, I found it useful to have both these files: in case `messages.gotext.json` was mess up, I always could refer to `out.gotext.json` as a source of truth. 

A possible case: `messages.gotext.json` should not be congruent to `out.gotext.json`, so I guess it will be handy  to copy only needed part of `out.gotext.json`

Reply:
We have git to fix a mess if we make it.

In what case only a copy of the file should be copied?

Reply:
> We have git to fix a mess if we make it.

not a git in this case but `make gen` - that take much more time then take a look at out.gotext.json in the next window split.

> In what case only a copy of the file should be copied?

For example when we do translation ahead
```
--- a/catalog/locales/en/messages.gotext.json
+++ b/catalog/locales/en/messages.gotext.json
@@ -25,6 +25,11 @@
             "id": "This method is not implemented yet.",
             "message": "This method is not implemented yet.",
             "translation": ""
+        },
+        {
+            "id": "TBD#1",
+            "message": "TBD#1",
+            "translation": "Some translation"
         }
     ]
 }

```

Reply:
>> We have git to fix a mess if we make it.
> not a git in this case but make gen 

I meant `git reset --hard` / `git checkout`.

Ahead of what? The actual code that uses that text? It's hard to image that situation in practice, TBH.

Let's keep one file in git – that way it is much harder to mess things up while holding them wrong.

Reply:
With `git reset --hard / git checkout` - you lose parsed/generated messages.
and if code base is big enough-  re-run `make gen` may take ~1-5 min - here is my point - but, ok, it can be worked around with other tools.

So, your suggestion is:
 - add to target gen `mv catalog/locales/en/{out,messages}.gotext.json`
 - remove `echo >> catalog/locales/en/out.gotext.json`
 - add `catalog/locales/en/out.gotext.json` to `.gitignore`

Correct?

Reply:
> you lose parsed/generated messages

No, if they are committed in git. That's the whole point of using it.

IIUC – yes. We can always make it more complex when/if we need it. Making things simpler is typically harder :)